### PR TITLE
latest changes

### PR DESCRIPTION
**Pull request by mm1ke**
This changes adds:
- tapdel: for deleting unused qtap devices
- save: for saving changes to a harddisk if vm starts in snapshotmode
- debug flag: output the vm settings and full start command
- updated 'network code': its possible to start vm without a network card and it should be easier now to add new network settings. Now you can set "bridge" and "none" as network-typ.

### CODE REVIEW

File: kvm.init [hunk 1]
```diff
@@ -42,6 +48,9 @@ PID_DIR="/var/run/kvm"
 # TMP dir
 TMP_DIR="/var/tmp"
 
+# 1=on, 0=off
+DEBUG=0
```
Comment (jlec) [inline kvm.init]: Would it be better to use string type values like "yes" or "true"? This would make the check later look nicer.

File: kvm.init [hunk 2]
```diff
@@ -67,6 +76,28 @@ scan_running_vm(){
 }
 
 #
+# tapdel()
+# removes unused tap devices
+#
+tapdel(){
```
Comment (jlec) [inline kvm.init:80]: You could add

local active_qtab

**Comment by jlec** [comment]
We should agree on an indention type. You are indenting with 2 spaces where I am indenting with tab. Personally I don't care what we are doing, but we should keep it consistent.

**Comment by mm1ke** [comment]
Hi Justin. I've changed Debug to true/false and added local vars. It's also clearly a good idea to keep indenting consistent. Actually i'm wondering what editor you use. I'm using kate with the default indenting settings, which means a tab is 2 spaces for me (usually)...
Anyway, i see my changes are now in this pull request too (didn't know that i added those changes to this pull request..). As you can see there are a few other changes/fixes too.

**Comment by jlec** [comment]
I am using nano, so every indention is manual.
So should we agree in 2 spaces then?

**Comment by mm1ke** [comment]
Yeah, it would be great if we could stay with 2 spaces :)

**Pull request closed without merging**